### DISCUSSION TITLE
Set up sessions count for testing HA Proxy limits

### DISCUSSION THREAD

**Discussion by ep4sh** [locustio/locust#2795] [Q&A]
Dear community,
please guide me how to set up sessions count via Locust to test my LoadBalancer limitations. I'm using HA Proxy as lb with user session limits:
- https://www.haproxy.com/documentation/haproxy-runtime-api/reference/set-rate-limit-sessions-global/

I've already tried to apply number of workers:
https://docs.locust.io/en/stable/running-distributed.html#expect-workers-number-of-workers

But the only thing I've got is an RPS increased.

Thanks!

**Comment by cyberw** [comment]
Hi!

Not 100% sure what is the question :) 

Do you want to increase the number of sessions until LB starts throwing old sessions out?

Probably you just need to increase the number of Users / ramp up speed further. Maybe clear cookies between each task iteration? https://docs.locust.io/en/stable/faq.html#clear-cookies-to-make-the-next-task-iteration-seem-like-a-new-user-to-my-system-under-test

**Reply by ep4sh** [reply]
Hi @cyberw, right, I want to increase the sessions count, apologize for confusion, and I didn't find how to apply it correctly (always getting 5 sessions).

**Reply by cyberw** [reply]
You get 5 sessions, using how many users? And what does your locustfile look like?
